nixos/modules/services/security/howdy/default.nix

```diff
@@ -0,0 +1,51 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+with lib; let
```

Comment:
again uses of `with` such as this are would be nice to have removed, `inherits` is usually a much better solution.